docs/src/content/docs/sdk.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Ultravox SDK"
+sidebar:
+  order: 0
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import SDKCards from '../../components/SDKCards.astro';
+
+The Ultravox [REST API](./api/auth/) is used to create calls but you must use one of the Ultravox client SDKs to join and end calls. This page uses examples in JavaScript. The concepts are the same across all the [SDK flavors](#sdk-flavors).
+
+## Ultravox Session
+The core of the SDK us the `UltravoxSession`. The session is used to join and leave calls.
+
+```javascript
+import { UltravoxSession } from 'ultravox-client';
+
+const session = new UltravoxSession();
+const state = await session.joinCall('wss://your-call-join-url');
+
+session.leaveCall();
+```
+
+## Session State
+When a call is joined, an `UltravoxSessionState` is returned. This object returns the current status, can be used to get text transcripts of the call, and surfaces debug messages that are helpful when building your application.
+
+### Status
+The session status is based on the `UltravoxSessionStatus` enum and can be one of the following:
+
+| status        | description                                 |
+|-------------- | ------------------------------------------- |
+| disconnecting | Session is in the process of disconnecting. |
+| connecting    | Session is establishing the connection.     |
+| idle          | Session is idle.                            |
+| listening     | Listening to the end user.                  |
+| thinking      | The model is processing/thinking.           |
+| speaking      | The model is speaking.                      |
+
+The status can be retrieved by adding an event listener to the state:
+
+```javascript
```

Comment:
If we were to show code for multiple clients, this would be a good place to start since event listening is different across implementations.

Reply:
yes makes sense. i will add adding more client examples to the list for later this week.